.github/workflows/CI.yml

```diff
@@ -25,115 +38,64 @@ jobs:
         with:
           toolchain: ${{ matrix.rust }}
           override: true
-      # atk
       # nightly
-      - name: "atk: tests nightly"
-        run: xvfb-run --auto-servernum cargo test --manifest-path atk/Cargo.toml --all-features
+      - name: tests nightly
+        run: xvfb-run --auto-servernum cargo test --manifest-path ${{ matrix.triple.name }}/Cargo.toml ${{ matrix.triple.nightly }}
```

Comment:
Can we run xvfb once as a `service` instead of wrapping each command?

Reply:
I think there are actions that does that for you and make sure to shut down the service once the command has stopped running